### PR TITLE
patch code to avoid committing against the database in GET endpoints

### PR DESCRIPTION
**Pull request by matheushent**
#### What
This PR applies a simple patch in the API code to avoid running `session.commit()` against the database while only read queries are run.

#### Why
Experimental fix for the performance issues currently on production

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Review by dusktreader — APPROVED** [review]

File: agent/pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "license-manager-agent"
-version = "3.0.9"
+version = "3.0.10-alpha.1"
```
Comment (julianaklulo) [inline agent/pyproject.toml]: Should this be `3.0.10a1`?

File: backend/pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "license-manager-backend"
-version = "3.0.9"
+version = "3.0.10-alpha.1"
```
Comment (julianaklulo) [inline backend/pyproject.toml]: Same here

File: lm-cli/pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "license-manager-cli"
-version = "3.0.9"
+version = "3.0.10-alpha.1"
```
Comment (julianaklulo) [inline lm-cli/pyproject.toml]: And here

File: agent/pyproject.toml (hunk 1 above)
Comment (matheushent) [reply agent/pyproject.toml]: there's no standard across our services actually. Some use this nomenclature, other use `ax`. Anyway for me, I will change

**Pull request merged**